src/rpp/rpp/operators/details/early_unsubscribe.hpp

```diff
@@ -16,9 +16,9 @@ namespace rpp::details
 {
 struct early_unsubscribe_state
 {
-    early_unsubscribe_state(rpp::composite_subscription subscription_of_subscriber) : childs_subscriptions(subscription_of_subscriber.make_child()) {}
+    early_unsubscribe_state(const composite_subscription& subscription_of_subscriber) : childs_subscriptions(subscription_of_subscriber.make_child()) {}
 
     // use this subscription as source for any chuld subscription that should be early unsubscribed
-    rpp::composite_subscription childs_subscriptions{};
+    composite_subscription childs_subscriptions{};
```

Comment:
Would you mind to use `git rebase` to move this changes to your first commit about the introduction of this helper class?

Reply:
I think no, because anyway PR would be squashed =)